### ISSUE TITLE
sub docs page in the documentation is missing the <code ..> tag

### ISSUE THREAD

**Issue by lineus** [Automattic/mongoose#6038]
http://mongoosejs.com/docs/subdocs.html# ( v.5.0.1 )

all of the code examples are hosed on this page for me, showing no formating.

![screenshot 2018-01-24 15 40 32](https://user-images.githubusercontent.com/6015453/35355973-168d52f2-011d-11e8-8717-09109f017a21.png)

**Comment by Mikhail-MM** [comment]
I am also seeing this in the Schema page docs

**Comment by lineus** [comment]
it looks like a few jade files in the docs are using a different syntax for code highlighting.
```
docs: grep '```' *jade | sort -u
acquit.jade:        ```
acquit.jade:        ```javscript
connections.jade:    ```
connections.jade:    ```javascript
documents.jade:    ```
documents.jade:    ```javascript
faq.jade:    ```
faq.jade:    ```javascript
guide.jade:    ```
guide.jade:    ```javascript
index.jade:    ```
index.jade:    ```javascript
middleware.jade:    ```
middleware.jade:    ```javascript
migration.jade:    ```
migration.jade:    ```javascript
plugins.jade:    ```
plugins.jade:    ```javascript
populate.jade:    ```
populate.jade:    ```javascript
queries.jade:    ```
queries.jade:    ```javascript
docs: grep ':js' *jade | sort -u
browser.jade:        :js
models.jade:  :js
schematypes.jade:  :js
subdocs.jade:  :js
docs: 
```

**Comment by lineus** [comment]
looks like this changed in [0e2231c7d162983bc4b4b4332cde8a2438ebf2ac](https://github.com/Automattic/mongoose/commit/0e2231c7d162983bc4b4b4332cde8a2438ebf2ac#diff-24bc312659458782a5d06139755cdf95) 
the pages that are still nested in <pre><code> tags are manually marked down with backticks.

**Comment by vkarpov15** [comment]
Thanks for reporting, fixed in 5.0.2
